tests/integration/fuzzing/fuzzing_test.py

```diff
@@ -98,7 +98,7 @@ async def test_max_fuzz_runs_less_or_equal_than_specified(
 async def test_strategies(
     run_cairo_test_runner: RunCairoTestRunnerFixture,
 ):
-    fuzz_max_examples = 10
+    fuzz_max_examples = 100
```

Comment:
It makes fuzz tests slower though. (@Radinyn complained about that.)

Reply:
Yes, but this is needed for correctness. With `10`, fuzzer wasn't able to explore large numbers :(

Reply:
I future PR I till chop this test into many ones, which only the tests that need such high max examples value will have one.